src/dbnode/persist/fs/read_write_test.go

```diff
@@ -253,6 +253,12 @@ func TestSimpleReadWrite(t *testing.T) {
 	readTestData(t, r, 0, testWriterStart, entries)
 }
 
+func TestCheckpointFileSizeBytesSize(t *testing.T) {
+	// These values need to match so that the logic for determining whether
+	// a checkpoint file is complete or not remains correct.
+	require.Equal(t, digest.DigestLenBytes, CheckpointFileSizeBytes)
```

Comment:
Instead of this, why don't you just make `CheckpointFileSizeBytes = digest.DigestLenBytes`?

Reply:
Yeah I discussed this with Prateek and we like the idea that the tests would bring if you changed one and not the other just to make the person think about the change more